### PR TITLE
Update orignal osquery package description

### PR DESCRIPTION
**Pull request by james-elastic**
## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by melissaburpo — COMMENTED** [review]
Looks good. I just had one comment/question.

File: packages/osquery/docs/README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 [`osqueryd`](https://osquery.readthedocs.io/en/latest/introduction/using-osqueryd/)
 in the JSON format. To set up `osqueryd` follow the osquery installation
 instructions for your operating system and configure the `filesystem` logging
 driver (the default). Make sure UTC timestamps are enabled.
 
+For a automatic `osqueryd` installation use the Osquery Manager integration. 
+
 ## Compatibility
 
 The  OSQuery integration was tested with logs from osquery version 2.10.2.
```
Comment (melissaburpo) [inline packages/osquery/docs/README.md:13]: I just noticed this one last reference to the original name. Should update here too: 
`OSQuery integration` --> `Osquery Log Collection`

**Review by melissaburpo — APPROVED** [review]

File: packages/osquery/docs/README.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
-# OSQuery Integration
+# Osquery Log Collection
```
Comment (mtojek) [inline packages/osquery/docs/README.md:1]: Could you please update the doc template file first https://github.com/elastic/integrations/blob/38a2b326c8b1c08294a5a59faa2c56e94e92194f/packages/osquery/_dev/build/docs/README.md and then build the integration? `elastic-package build`

File: packages/osquery/changelog.yml [hunk 3]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.1.2"
```
Comment (mtojek) [inline packages/osquery/changelog.yml:2]: I suppose this one may need formatting (`elastic-package format`).

**Review by mtojek — COMMENTED** [review]
Conceptually this change looks absolutely valid. Please just address posted comments.

**Comment by james-elastic** [comment]
Thanks @mtojek I used elastic-package to build / format. :)

**Review by mtojek — APPROVED** [review]
Errors are unrelated to this PR. You should be good to go.

**Comment by james-elastic** [comment]
Thanks @mtojek, I don't have the permission to merge this, could you do it for me? Thanks!

**Pull request merged**